RecoMET/METFilters/plugins/EcalDeadCellTriggerPrimitiveFilter.cc

```diff
@@ -158,6 +162,9 @@ class EcalDeadCellTriggerPrimitiveFilter : public edm::EDFilter {
 
   bool useTPmethod_, useHITmethod_;
 
+  bool exclusivelykTPSaturated_;
+  bool considerkTPSaturated_;
```

Comment:
Hi Slava,

During development these names were used before the options useTTsum_ and usekTPSaturated_ were added. These two lines were left behind when the code was cleaned up in preparation for the PR. They can be removed, and I've updated the PR with them removed.

Regards,
Ken Call
________________________________
From: Slava Krutelyov [notifications@github.com]
Sent: Wednesday, February 24, 2016 6:28 AM
To: cms-sw/cmssw
Cc: Call, Ken
Subject: Re: [cmssw] Adds option to EcalDeadCellTriggerPrimitiveFilter to consider kTPSaturated flag (#13429)


In RecoMET/METFilters/plugins/EcalDeadCellTriggerPrimitiveFilter.cc<https://github.com/cms-sw/cmssw/pull/13429#discussion_r53937346>:

> @@ -158,6 +162,9 @@ class EcalDeadCellTriggerPrimitiveFilter : public edm::EDFilter {
>
>    bool useTPmethod_, useHITmethod_;
>
> +  bool exclusivelykTPSaturated_;
> +  bool considerkTPSaturated_;


I'm not sure why these two are added: there are not used in the class methods.
Please clarify.

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/13429/files#r53937346>.